<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmithWidget</class>
 <widget class="QWidget" name="SmithWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmithWidget</string>
  </property>
  <widget class="QWidget" name="widget_paint" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>410</width>
     <height>410</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>410</width>
     <height>410</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>410</width>
     <height>410</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>显示参数</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>归一化导纳</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_y">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>电长度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_l">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>驻波系数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_vswr">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>反射系数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_gamma">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>模长</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_abs">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>相位</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_angle">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>相关参数输入</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>负载阻抗 Zl </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_zl_r">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_zl_i">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>j</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>特征阻抗 Z0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_z0">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>计算归一化阻抗</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>归一化阻抗</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_z_r">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_z_i">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>j</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>输入阻抗计算</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>传输线长度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_l0">
      <property name="text">
       <string>0.25</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>λ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>输入阻抗</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_zi">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>颜色标注</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>反射系数圆</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>电阻圆</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>电抗圆</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(0, 85, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>负载阻抗</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_25">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>输入阻抗</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
